<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nullstellenfinder</class>
 <widget class="QMainWindow" name="Nullstellenfinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nullstellenfinder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:black
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="ButtonMenu">
      <property name="toolTip">
       <string>geht wieder zurück zur Rechnerauswahl</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(54, 54, 54)</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextBrowser" name="textNullstellen">
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="placeholderText">
       <string>Hier stehen dann die Nullstellen</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="label_Equation">
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Hier erscheint dann die Gleichung</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="ButtonBerechnen">
      <property name="styleSheet">
       <string notr="true">background:rgb(0, 0, 255)</string>
      </property>
      <property name="text">
       <string>Nullstellen Berechnen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineKoeffizienten">
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="placeholderText">
       <string>Hier die Koeffizienten eingeben</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="text">
       <string>Bitte die Koeffizienten durch ein Komma getrennt eingeben. Angefangen bei dem Monom mit dem höchsten Grad.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
